ansible/roles/test/tasks/qos_sai.yml

```diff
@@ -37,14 +37,18 @@
         - lldpd
         - lldp-syncd
 
-    - name: Disable bgpd
+    - name: Ensure BGP Daemon stopped
       become: yes
-      lineinfile: dest=/etc/quagga/daemons
-                  regexp=^bgpd=.*$
-                  line='bgpd=no'
-      notify:
-        - Restart Quagga Daemon
```

Comment:
does this need to be removed?

Reply:
@neethajohn Yes, once we stop the process we don't need to restart service.